apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx

```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
 
 </header>
 
+After [successfully deploying and creating the repository and live preview of your store](/docs/getting-started/1-faststore-onboarding/onboarding#repository-and-live-store-created), 
+you can access the Dashboard page for detailed information about your store deploys. 
+The Dashboard displays the current status of your store, including its domains, deployed versions, store preview, and commits.
+
+![dashboard-overview](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-overview___56923825cfdcfc7018cbfe3a9aa726ee.gif)
+
+On the Dashboard page, you will find two main views: Overview and Deploys. For more information about each view, continue reading the next sections.
+
+---
+
+## Overview
+
+The **Overview** view provides information about the production deployment of your store.
+
+![overview-view](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-view-overview___7b8f9c67ff14be86263514d62ae471f1.png)
+
+
+Check out the table below for more information:
+
+
+| Field                 | Description                                                                                                                           |
+| :-------------------- | :------------------------------------------------------------------------------------------------------------------------------------ |
+| **Preview FastStore** | Opens a preview of your store in a new tab.                                                                                           |
+| **Deployment**        | Opens the deployed version of the store in a new tab.                                                                                 |
+| **Domain**            | Opens the URL of your store's domain, which is the same as the preview.                                                               |
+| **Last commit**       | Displays the most recent commit made to your store's repository, including the date, month, year, and the person who made the commit. |
+| **Status**            | Displays the status of the deployment.                                                                                                |
+
+The **Status** field can have one of the following values: 
```

Comment:
We've just added a new status `Skipped`. Can you add it here pls?

More details on this PR: https://github.com/vtex-sites/faststore.admin/pull/41